Notebooks/LineOfSightEffects/line-of-sight_effects_tutorial.ipynb

```diff
@@ -0,0 +1,1542 @@
+{
```

Comment:
<p>do you need this block in the tutorial (even before actually demonstrating the features)? The error message returned look quite good themselves, even without the tutorial. Also, if you try to run all of the notebook, it gets interrupted by this error message</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I removed the code cells which generate the message and just left the text mentioning when the errors occur.</p>